latte/en/template-inheritance.texy

```diff
@@ -4,7 +4,7 @@ Template Inheritance and Reusability
 .[perex]
 Template reusability and inheritance mechanisms are here to boosts your productivity because each template contains only its unique contents and the repeated elements and structures are reused. We introduce three concepts of inheritance: [layout|#Layout Inheritance], [horizontal|#Horizontal Inheritance] and [unit|#Unit Inheritance].
 
-The concept of Latte template inheritance is similar to PHP class inheritance. You define a **parent template** that other **child templates** can extend from and can override parts of the parent template. It works great when elements share a common structure. Sounds complicated? Don't worry, it's not.
+The concept of Latte template inheritance is similar to PHP class inheritance. You define a **parent template** that other **child templates** can extend from and can override parts of the parent template. It works great when elements share a common structure. Sounds complicated? Don't worry, it is very basic.
 
```

Comment:
Are you sure here? ;-)

Reply:
No, it just sound weird to me. :-) If you searched for opposite to `complicated`, then I think `easy` or `simple` fits better.

Reply:
Aha, now I understand, what is the point. Well, when updating docs I work only in GitHub UI so once I find button for rebase this is fixed. :-)

Reply:
Ok, I've had to do it manually.